### DISCUSSION TITLE
"Generate Suggestions" should add to the Labeling Suggestions, not replace them

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#836] [Ideas]
Hello,

Is it possible for the new frame suggestions to be added to the list, rather than to replace them? With the new functionality from https://github.com/talmolab/sleap/issues/627 (thank you!!), it would be convenient when adding a set of ~20 new videos to move to each video, "Generate Suggestions" with y samples per video, and then move to the next video and repeat until I have 10*y samples total, and then run inference on suggested frames.

Currently, the "Generate Suggestions" function deletes all frames in the Labeling Suggestions list, and then replaces them with the new suggestions. Therefore my current my process is to run inference 20 different times if I want to add videos and correct predictions on them inside a much larger dataset.

Thanks!

**Comment by roomrys** [comment]
This was completed in #874!
